platforms/android/src/jp/wizcorp/phonegap/plugin/WizAssets/WizAssetManager.java

```diff
@@ -96,18 +98,18 @@ public JSONObject getAllAssets() {
         }
         return returnObject;
     }
-    
+
     private void buildDB() {
         // Init DB from bundle assets out to storage
         Log.d(TAG, "Init DB");
-        
+
         // Open the .db file from assets directory
         try {
             InputStream is = that.getAssets().open(DATABASE_INTERNAL_FILE_PATH + DATABASE_NAME);
 
             // Copy the database into the destination
             OutputStream os = new FileOutputStream(DATABASE_EXTERNAL_FILE_PATH + File.separator + DATABASE_NAME);
-        
+
             byte[] buffer = new byte[1024];
```

Comment:
I can almost guarantee it's a very unfortunate size. Rather than benchmarking, I would suggest researching best practices. They must exist.